clang/test/CodeGenSYCL/uses_aspects.cpp

```diff
@@ -28,7 +28,7 @@ template <sycl::aspect Aspect>
 void func5() {}
 
 [[__sycl_detail__::__uses_aspects__(sycl::aspect::cpu)]] void func6();
-// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !sycl_used_aspects ![[ASPECTS4:[0-9]+]] {
+// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !sycl_used_aspects ![[ASPECTS1]] {
```

Comment:
Same here. Please change command line to remove -disable-llvm-passes instead of changing  what this test verifies